cfgov/v1/forms.py

```diff
@@ -321,7 +321,7 @@ def get_page_set(self):
         else:
             query = self.generate_query()
             return self.filterable_pages.filter(query).distinct().order_by(
-                '-date_published'
+                '-initial_filing_date'
```

Comment:
This was a copy paste error that got introduced and needs to be cleaned up to preserve existing EnforcementAction filterable list search.